Dockerfile

```
@@ -8,8 +8,8 @@ RUN export DEBCONF_NONINTERACTIVE_SEEN=true \
            DEBIAN_PRIORITY=critical \
            TERM=linux ; \
     apt-get -qq update ; \
-    apt-get -yyqq upgrade ; \
-    apt-get -yyqq install ca-certificates libcap2-bin; \
+    apt-get -qq upgrade ; \
+    apt-get -qq install ca-certificates libcap2-bin; \
```

```suggestion
    apt-get -qq --no-install-recommends install ca-certificates libcap2-bin; \
```
might be worth adding this option as well, then it won't install https://packages.debian.org/sid/libpam-cap

---

Done 👍